CMakeLists.txt

```diff
@@ -138,7 +138,7 @@ if (${FAASM_PERF_PROFILING})
     # Don't omit frame pointers (lets perf get better stack traces)
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fno-omit-frame-pointer")
 else ()
-    find_package(LLVM REQUIRED CONFIG)
+    find_package(LLVM 10 REQUIRED CONFIG)
```

Comment:
I needed this as I have various `LLVM` versions in my local machine. I think it's a good idea to pin versions.